ecl/ecl-bundle/ecl-bundle.cpp

```diff
@@ -379,7 +379,7 @@ class CBundleInfo : public CInterfaceOf<IBundleInfo>
                 if (!path.length())
                     path.append(".");
                 if (bundleFile->isDirectory()==fileBool::foundYes && !directoryContainsBundleFile(bundleFile))
-                    includeOpt.appendf(" \"-I%s\"", bundle);
+                    includeOpt.appendf(" \"-I%s\"", cleanedParam.str());
                 else
                     includeOpt.appendf(" \"-I%s\"", path.str());
```

Comment:
@richardkchapman The code is actually going into the else branch - so it is still failing.